resources/assets/js/features/tripsList/components/MapModal.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux';
+import { hideModal } from "../actions";
+import MapDirection from './MapDirection';
+
+import './css/modal.scss';
+
+class MapModal extends Component {
```

Comment:
use already define Modal component, take a look at how reset password modal is made